components/annotationList.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import Router from "next/router";
+import { render } from "react-dom";
+
+/**
+ * annotations(annArray) : function that returns 1 array of all the annotation tags
+ *
+ * @param {*} annArray : array of annotations (which each annotation contains an array of tags)
+ */
+function annotations(annArray) {
+  if (Array.isArray(annArray)) {
+    var returnArray = [];
+    return annArray.map(x => returnArray.concat(x.Tags)).flat();
```

Comment:
hmmm this looks like more complicated than it should be.
I think what we want to do is to reduce the arrays inside an object to one array, right? map is not intended for such usage. try reducer 
` annArray.reduce((initialArray, obj) => initialArray.concat(obj.Tags),[]);`